CMakeLists.txt

```diff
@@ -67,6 +67,10 @@ macro(build_lib_name LIB_VAR NAME)
       ${CMAKE_STATIC_LIBRARY_PREFIX}${NAME}${CMAKE_STATIC_LIBRARY_SUFFIX})
 endmacro()
 
+if(ENABLE_IOTJS_EXTERNAL_CONTEXT)
+  iotjs_add_compile_flags(-DIOTJS_ENABLE_EXTERNAL_CONTEXT)
+endif()
```

Comment:
Shouldn't this setting also be reported by `cmake/iotjs.cmake`  (around `message("IoT.js configured with:")`)?